lib/models/adapty_android_subscription_update_params.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:core';
+
+import 'package:adapty_flutter/models/adapty_enums.dart';
+import 'package:flutter/foundation.dart';
+
+class AdaptyAndroidSubscriptionUpdateParams {
+  /// The product id for current subscription to change.
+  late String _oldSubVendorProductId;
+
+  /// The proration mode for subscription update.
+  /// The possible values are: immediateWithTimeProration, immediateAndChargeProratedPrice, immediateWithoutProration, deferred, immediateAndChargeFullPrice.
+  late AdaptyAndroidSubscriptionUpdateProrationMode _prorationMode;
+
+  AdaptyAndroidSubscriptionUpdateParams(String oldSubVendorProductId,
+      AdaptyAndroidSubscriptionUpdateProrationMode prorationMode) {
+    this._oldSubVendorProductId = oldSubVendorProductId;
+    this._prorationMode = prorationMode;
+  }
+
+  Map<String, String> get map {
```

Comment:
Usual Dart convention would be to make this a method i.e. `toMap()`, as it's not returning a property of the class. More converting the format of the object.

This can be shortened to
```dart
Map<String, String> toMap() => {
  'old_sub_vendor_product_id': _oldSubVendorProductId,
  'proration_mode': describeEnum(_prorationMode)
};
```

Reply:
@danieljimeneznz thank you, fixed